From Thierry Vignaud <tvignaud@mandriva.com>  (Mandriva)

We now lacks /usr/include/linux/pci_ids.h which break ldetect build...
Can you readd it please?
Thanks
---
 include/linux/Kbuild |    1 +
 1 file changed, 1 insertion(+)

--- linux-2.6.36-rc6-git2-mnb0.2/include/linux/Kbuild.include-kbuild-export-pci_ids.orig	2010-09-29 10:30:05.000000000 +0300
+++ linux-2.6.36-rc6-git2-mnb0.2/include/linux/Kbuild	2010-10-03 12:31:06.884004076 +0300
@@ -277,6 +277,7 @@ header-y += param.h
 header-y += parport.h
 header-y += patchkey.h
 header-y += pci.h
+header-y += pci_ids.h
 header-y += pci_regs.h
 header-y += perf_event.h
 header-y += personality.h
